packages/menu/src/MenuGroup/MenuGroup.tsx

```diff
@@ -14,10 +27,54 @@ import { MenuGroupProps } from './MenuGroup.types';
  * @param props.children Content to appear inside of the MenuGroup.
  *
  */
-export function MenuGroup({ children, className, ...rest }: MenuGroupProps) {
+export function MenuGroup({
+  children,
+  className,
+  title,
+  glyph,
+  ...rest
+}: MenuGroupProps) {
+  const { theme, darkMode } = useMenuContext();
+  const id = useIdAllocator({ prefix: 'lg-menu-group' });
+  const { depth } = useMenuGroupContext();
```

Comment:
Be default, this will be `0`, if `MenuGroup` is not nested?

Reply:
Yep, 
```
/// packages/menu/src/MenuContext/GroupContext.tsx
export const MenuGroupContext = createContext<MenuGroupContextData>({
  depth: 0,
  hasIcon: false,
});
```